include/crow/app.h

```diff
@@ -161,7 +161,16 @@ namespace crow
         ///Run the server
         void run()
         {
+#ifndef CROW_DISABLE_STATIC_DIR
+            route<crow::black_magic::get_parameter_tag("/static/<path>")>("/static/<path>")
```

Comment:
Maybe extract `static` to const and allow user to change it on compile time?

Reply:
I'll look into it

Reply:
I'm not sure how the user can modify a const on compile time, is there a way of setting a const on compile time without say macros or some cast?

Reply:
Sorry, my bad. I thought about changing this variable before merging all includes, but that's bad idea. Maybe some configuration property?

Reply:
it's possible to add a macro like `CROW_LOG_LEVEL` in `settings.h`, but right now all it takes the user to set their own static directory is to set the disable flag and create a 2 line route. I'm fine with both adding a macro or not changing anything :)

Reply:
Ok, so we can stay with your solution :)

Reply:
oh man, I just made it work with macros 😅 XD